### PR TITLE
Create 2017-09-03-variables-in-inspector.md

### PR DESCRIPTION
**Pull request by thomasburt255**
Added a post on setting variables in the inspector.

### CODE REVIEW

File: _posts/2017-09-03-variables-in-inspector.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+date:   2017-09-03
+categories: Tips&Tricks
+tags: inspector code beginner
+excerpt:
+mathjax: false
+author: BaconGamer88
+---
+
+Beginner programmers in Duality may have begun to create their scripts, only to wonder why they can't read/write variables in the 
+inspector. In Unity, you would declare the variable as public or put `[SerializeField]` in front of the variable. However, both approaches
```
Comment (mfep) [inline _posts/2017-09-03-variables-in-inspector.md:13]: If we mention serializing, why don't we explain Duality's `[DontSerialize]` attribute?

File: _posts/2017-09-03-variables-in-inspector.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+tags: inspector code beginner
+excerpt:
+mathjax: false
+author: BaconGamer88
+---
+
+Beginner programmers in Duality may have begun to create their scripts, only to wonder why they can't read/write variables in the 
+inspector. In Unity, you would declare the variable as public or put `[SerializeField]` in front of the variable. However, both approaches 
+will not work in Duality. So how would you go about doing this? Well, this snippet shows how to do this with a float variable, but it will 
+work with any variable type, including integers, strings and booleans.
```
Comment (mfep) [inline _posts/2017-09-03-variables-in-inspector.md]: I might add here, that any non-builtin types can be exposed too. (I know that logically "any variable type" covers these, but it's a bit more straightforward. Or we can omit the "including" part.)

File: _posts/2017-09-03-variables-in-inspector.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+```csharp
+public class EpicScript : Component
```
Comment (mfep) [inline _posts/2017-09-03-variables-in-inspector.md]: Naming variables 'epic' has quite low humor content, but renders the whole article a bit immature. I think we should call stuff something boring, like `Foo` or `MyComponent`.

File: _posts/2017-09-03-variables-in-inspector.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    // Declare a property to access it for reading and writing.
+    // The editor will use this in the Object Inspector.
+    public float epicValue
+    {
+        get { return this.epicValue; }
+        set { this.epicValue = value; }
+    }
+    // Insert following code here.
+```
+This code tells the inspector to allow the user to modify/read the variable through the inspector and then gets the value set in the
```
Comment (mfep) [inline _posts/2017-09-03-variables-in-inspector.md]: The closing sentence is somewhat convoluted. Wouldn't it be simpler if we just stated that the `get` and `set` methods are called when we modify the value in the inspector?

**Review by mfep — CHANGES_REQUESTED** [review]
Thanks for the article! Good to see stuff already happening to the repo. I've added some comments to consider, but obviously I'm open to debate.

**Comment by thomasburt255** [comment]
@mfep Thanks for the feedback. This is my first ever github project I've contributed to, so sorry for the amateur quality. I shall get right to work!

**Comment by thomasburt255** [comment]
@mfep Have added all suggested changes. Please review.

**Comment by mfep** [comment]
Since my last comment, I kind of changed my mind, and I think we should write an entire 'Tip' about serialization. So let's omit that part for now. Other than that, I'd merge this, thanks for the PR!

**Review by mfep — APPROVED** [review]

**Comment by mfep** [comment]
Since, there's been no activity here for a while, I'm merging this, and do the little change requested.

**Pull request merged**